### PR TITLE
CSEN-24 Response entities was implemented

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
@placccebo

File: src/main/scala/com/bwsw/cloudstack/entities/responses/Entity.scala [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package com.bwsw.cloudstack.entities.responses
+
+class EntityResponse(entityList: EntityList)
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/entities/responses/Entity.scala]: add a general description of this classes

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/main/scala/com/bwsw/cloudstack/entities/common/JsonMapper.scala [hunk 2]
```diff
@@ -27,6 +27,9 @@ import com.fasterxml.jackson.databind.{DeserializationFeature, ObjectMapper}
 import com.fasterxml.jackson.module.scala.DefaultScalaModule
 import org.slf4j.LoggerFactory
 
+/**
+  * Class is responsible for conversion from entities to json string and from json string to entities
+  */
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/entities/common/JsonMapper.scala]: This mapper provides functionality to convert your model objects (classes and structs) to and from json string.

File: src/main/scala/com/bwsw/cloudstack/entities/Executor.scala [hunk 3]
```diff
@@ -29,13 +29,25 @@ import org.slf4j.LoggerFactory
 
 import scala.util.{Failure, Success, Try}
 
+/**
+  * Class is responsible for interaction with CloudStack
+  *
+  * @param settings see: Executor.Settings
+  * @param waitIfServerUnavailable If true and the server is unavailable, send requests until available.
+  *                                If false, throw the exception.
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/entities/Executor.scala]: an exception

File: src/main/scala/com/bwsw/cloudstack/entities/Executor.scala [hunk 4]
```diff
@@ -29,13 +29,25 @@ import org.slf4j.LoggerFactory
 
 import scala.util.{Failure, Success, Try}
 
+/**
+  * Class is responsible for interaction with CloudStack
+  *
+  * @param settings see: Executor.Settings
+  * @param waitIfServerUnavailable If true and the server is unavailable, send requests until available.
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/entities/Executor.scala]: If true and the server is unavailable, sends requests until the server has become available

File: src/main/scala/com/bwsw/cloudstack/entities/Executor.scala [hunk 5]
```diff
@@ -29,13 +29,25 @@ import org.slf4j.LoggerFactory
 
 import scala.util.{Failure, Success, Try}
 
+/**
+  * Class is responsible for interaction with CloudStack
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/entities/Executor.scala]: Class provides functionality to interact with CloudStack

File: src/main/scala/com/bwsw/cloudstack/entities/Executor.scala [hunk 6]
```diff
@@ -72,5 +84,13 @@ class Executor(settings: Executor.Settings,
 }
 
 object Executor {
+  /**
+    * Class is responsible for providing settings for interaction with CloudStack
+    *
+    * @param endpoints array with endpoints of CloudStack server
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/entities/Executor.scala]: array of

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/main/scala/com/bwsw/cloudstack/entities/Executor.scala (hunk 4 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/entities/Executor.scala]: ok

File: src/main/scala/com/bwsw/cloudstack/entities/Executor.scala (hunk 3 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/entities/Executor.scala]: ok

File: src/main/scala/com/bwsw/cloudstack/entities/Executor.scala (hunk 5 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/entities/Executor.scala]: ok

File: src/main/scala/com/bwsw/cloudstack/entities/Executor.scala (hunk 6 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/entities/Executor.scala]: ok

File: src/main/scala/com/bwsw/cloudstack/entities/common/JsonMapper.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/entities/common/JsonMapper.scala]: ok

**Review by placccebo — APPROVED** [review]

**Pull request merged**